### DISCUSSION TITLE
OpenVPN in bridged mode per Gateway?

### DISCUSSION THREAD

**Discussion by boris1000** [RaspAP/raspap-webgui#1100] [Q&A]
I am not an expert, but would like to use RaspAP as VPN Gateway. 
Currently I am using Cyberghost as provider and they also do provide an .ovpn file.  
Now I am reading when using bridged mode clients won´t use the VPN when connected to RaspAP, is that really true?
I planned to only use connections via LAN (not WLAN) means assigning the clients per Gateway to RaspAP.
Will that work with VPN as well?
Do I just need to switch off WLAN and RaspAP works still as Gateway in bridged mode for the clients or how does that work?

**Comment by billz** [comment]
RaspAP may be used as a WLAN VPN gateway (OpenVPN or WireGuard) or a bridge, but not both simultaneously, nor restricted to LAN clients. You might find some tutorials online to create this setup, but it's not supported by this project.

**Comment by boris1000** [comment]
Thanks. So this means I need to search for setting up RaspAP as a bridge
for Lan clients? I'm asking cause I'm not an expert. Is this possible using
the GUI from RaspAP only and maybe you have a link for such a description?

**Reply by billz** [reply]
Short answer: no.

RaspAP is focused primarily on administering WLANs (hence, "AP"). In bridged AP mode, DHCP is delegated to your router, so these functions are disabled in the UI.

Your goals are outside the scope of this project, so you will need to look for an alternative.
